pypeman/channels.py

```diff
@@ -356,19 +361,21 @@ async def handle(self, msg):
                     await self.join_nodes[0].handle(result.copy())
                 return result
             except Dropped as exc:
+                self.logger.info("%s DROP msg %s", self, msg)
                 msg.chan_exc = exc
                 msg.chan_exc_traceback = traceback.format_exc()
                 await self.message_store.change_message_state(msg_store_id, message.Message.PROCESSED)
                 if self.drop_nodes and not has_callback:
                     await self.drop_nodes[0].handle(msg.copy())
-                raise
+                raise  # TODO: maybe add a param to raise exc or not ?
```

Comment:
I'm not sure I have enough context.

What would be the use case to propagate the `Dropped` exception?

What would be the usecase to **not** propagate the `Dropped` exception?